src/main/resources/liquibase/scripts/user.sql

```diff
@@ -0,0 +1,11 @@
+--liquibase formatted sql
+
+
+--changeset ependi: 1
+CREATE TABLE notification_task(
+    id      integer                 not null,
+    chatId  integer                 not null,
+    message text                    not null,
+    date    date                    not null,
+    time    time                    not null
+)
```

Comment:
```suggestion
CREATE TABLE notification_task(
    id        int8      not null,
    chat_id   int8      not null,
    message   text      not null,
    data_time timestamp not null
)
```
- В сущности для хранения идентификаторов используется Long. Поэтому в таблице нужно использовать int8.
- chatId нужно переименовать в chat_id
- Не нужно разделять дату и время. Под них заводим одну колонку data_time.
- Тип data_time должен быть timestamp